src/ToDoItem.js

```diff
@@ -5,8 +5,15 @@ const ToDoItem = ({ item, onCompleteItem }) => {
     const isOverDue = new Date(due) <= new Date();
 
     return (
-        <tr className={isOverDue ? 'overdue' : 'notDueYet'}>
-            <td><input type='checkbox' checked={completed} onChange={() => onCompleteItem(id)} /></td>
+        <tr
+            className={isOverDue ? 'overdue' : 'notDueYet'}>
```

Comment:
No need for new line here.
```suggestion
        <tr className={isOverDue ? 'overdue' : 'notDueYet'}>
```